front/app/containers/UsersShowPage/UserHeader.tsx

```diff
@@ -82,98 +37,78 @@ const Bio = styled.div`
-
-export const UserHeader = memo<Props>((props) => {
-  const { user, authUser, windowSize } = props;
-
+const UserHeader = ({ userSlug }: Props) => {
+  const authUser = useAuthUser();
+  const user = useUser({ slug: userSlug });
+  const isTablet = useBreakpoint('tablet');
   const hideBio = useFeatureFlag({ name: 'disable_user_bios' });
 
-  const smallerThanSmallTablet = windowSize
-    ? windowSize <= viewportWidths.tablet
-    : false;
-
-  if (!isNilOrError(user)) {
-    const memberSinceMoment = moment(user.attributes.created_at).format('LL');
-    let hasDescription = false;
-
-    forOwn(user.attributes.bio_multiloc, (value, _key) => {
-      if (!isEmpty(value) && value !== '<p></p>' && value !== '<p><br></p>') {
-        hasDescription = true;
-      }
-    });
-
-    return (
-      <Container>
-        <UserAvatar>
-          <Avatar userId={user.id} size={smallerThanSmallTablet ? 120 : 150} />
-        </UserAvatar>
-
-        <UserInfo>
-          <FullName id="e2e-usersshowpage-fullname">
-            {user.attributes.first_name} {user.attributes.last_name}
-          </FullName>
-          <JoinedAt>
-            <FormattedMessage
-              {...messages.memberSince}
-              values={{ date: memberSinceMoment }}
-            />
-          </JoinedAt>
-          {!hideBio &&
-            !isEmpty(user.attributes.bio_multiloc) &&
-            hasDescription && (
-              <Bio>
-                <QuillEditedContent>
-                  {user.attributes.bio_multiloc && (
-                    <T
-                      value={user.attributes.bio_multiloc}
-                      supportHtml={true}
-                    />
-                  )}
-                </QuillEditedContent>
-              </Bio>
-            )}
-          {!isNilOrError(authUser) && authUser.id === user.id && (
-            <EditProfileButton
-              linkTo="/profile/edit"
-              buttonStyle="text"
-              icon="edit"
-              className="e2e-edit-profile"
-              bgHoverColor={colors.background}
-            >
-              <FormattedMessage {...messages.editProfile} />
-            </EditProfileButton>
-          )}
-        </UserInfo>
-      </Container>
-    );
+  if (isNilOrError(user)) {
+    return null;
   }
-
-  return null;
-});
-
-const Data = adopt<DataProps, InputProps>({
-  windowSize: <GetWindowSize />,
-  authUser: <GetAuthUser />,
-  user: ({ userSlug, render }) => <GetUser slug={userSlug}>{render}</GetUser>,
-});
-
-const WrappedUserHeader = (inputProps: InputProps) => (
-  <Data {...inputProps}>
-    {(dataProps) => <UserHeader {...inputProps} {...dataProps} />}
-  </Data>
-);
-
-export default WrappedUserHeader;
+  const memberSinceMoment = moment(user.attributes.created_at).format('LL');
+  let hasDescription = false;
+
+  forOwn(user.attributes.bio_multiloc, (value, _key) => {
+    if (!isEmpty(value) && value !== '<p></p>' && value !== '<p><br></p>') {
+      hasDescription = true;
```

Comment:
I know this was here before, but would !isEmpty(value) not cover the `<p></p>` and `<p><br></p>` cases? :thinking:

Reply:
I think the isEmpty helper probably just checks for an empty string and the additional checks make sure you don't submit empty content even if there is technically a `p` in the html.